eslint-config-5minds-typescript/README.md

```diff
@@ -0,0 +1,73 @@
+# ESLint config for 5Minds TypeScript projects
+
+- install with
+  ```
+  npm install --save-dev eslint-config-5minds-typescript
+  ```
+- install the necessary plugins and parsers with
+  ```bash
+  npm install --save-dev \
+    @typescript-eslint/parser \
+    @typescript-eslint/eslint-plugin \
+    eslint-plugin-6river \
+    eslint-plugin-import \
+    eslint-plugin-no-null
```

Comment:
Ich hab gerade ein Install mit dem Befehl hier versucht und folgende Meldungen bekommen: 

![Bildschirmfoto 2019-05-02 um 11 39 19](https://user-images.githubusercontent.com/15343316/57067705-0a1a3d80-6cd0-11e9-8405-b539b2c425e6.png)


1. Dass `eslint@^5.0.0` eine peer-Dependency ist, sollte mMn auch hier in der Readme erwähnt werden.
2. Was ist mit `eslint-plugin-jsx-a11y` und `eslint-plugin-react`?
3. Wenn `tslint-eslint-config-5minds` und `tslint-config-5minds` wirklich von dem Plugin hier gebraucht werden, dann sollten die noch für den Umgang mit typescript@3 aktualisiert werden.

Reply:
Wenn ich eslint nun als Peer Dependency angebe, bekomm ich nur noch die her: 

![Bildschirmfoto 2019-05-02 um 12 08 32](https://user-images.githubusercontent.com/15343316/57068795-105de900-6cd3-11e9-83d3-7fd434b279c9.png)